lib/tunneler_tcp.c

```diff
@@ -236,7 +261,7 @@ u8_t recv_tcp(void *tnlr_ctx_arg, struct raw_pcb *pcb, struct pbuf *p, const ip_
     u16_t src_p = lwip_ntohs(tcphdr->src);
     u16_t dst_p = lwip_ntohs(tcphdr->dest);
 
-    ZITI_LOG(DEBUG, "received segment %s:%d->%s:%d",
+    ZITI_LOG(TRACE, "received segment %s:%d->%s:%d",
```

Comment:
maybe this should be behind an compiler directive? for 'hot path' stuff like this even this is work that we might not want/need?

Reply:
Are you suggesting wrapping this and probably other logging statements in an #ifdef DEBUG or something similar? If so, that would prevent ordinary folk (tunneler app users) from increasing the log level unless they have a debug build of the app in their possession. If this is the idea then I'd rather first determine that the performance hit is worth the reduced supportability.